File: LV0/간단한 식 계산하기.js
function solution(binomial) {
  binomial = binomial.split(" ");
  let first = parseInt(binomial[0]);
  let second = binomial[1];
  let third = parseInt(binomial[2]);
  switch (second) {
    case "+":
      return first + third;
    case "-":
      return first - third;
    case "*":
      return first * third;
    case "/":
      return first / third;
  }
}

File: LV0/간단한 식 계산하기.py
def solution(binomial):
    binomial = binomial.split(" ")
    first = int(binomial[0])
    second = binomial[1]
    third = int(binomial[2])
    if second == "+":
        return first + third
    elif second == "-":
        return first - third
    elif second == "*":
        return first * third
    elif second == "/":
        return first / third
